DAO.sol

```diff
@@ -484,6 +490,8 @@ contract DAO is DAOInterface, Token, TokenSale {
         if (now < p.votingDeadline  // has the voting deadline arrived?
             // Have the votes been counted?
             || !p.open
+            // is the recipient still on the whitelist?
+            || isRecipientAllowed(p.recipient)
```

Comment:
so in this case, the voters would be blocked forever and the proposal deposit could never be returned.